### PR TITLE
Reduces Skeleton Speed

### PR DESCRIPTION
**Pull request by Megamaw**
Hell, hope I did it right this time.

All this does is change the speed modifier of skeletons from 1.5 to 1.2.

If there's an issue, some 'i' I haven't dotted or 't' I haven't crossed, please let me know.

Thank you all for being patient with me. I'm still figuring this out.

### CODE REVIEW

**Comment by DamianX** [comment]
why?

**Comment by Megamaw** [comment]
pomf said that currently as it stands, skeletons are unacceptably powerful to be a roundstart race.

so i had to rethink how fast i really made them.

by dropping their speed modifier, i was hoping to draw then closer in line to other nonhuman roundstart races-- an upside that doesn't completely overshadow the downsides.

**Comment by SonixApache** [comment]
>trying to get skeletons back as roundstart race


*get out*

**Comment by kilozombie** [comment]
I respect you  wanting the thing you had back again, since you clearly enjoyed it. But this isn't really the way to go about it.
Change hearts and minds; don't bargain.

**Comment by Zennyization** [comment]
are we all not technically playing as skeletons already?

**Comment by Diavill** [comment]
i have come here to laugh at you part deux

**Comment by Riekland** [comment]
didn't you make it to 1.5 times speed like literally a week ago or did I miss something
anyway they should be faster but have a random chance to break their legs while running unless they have milk in their system because of their frail space bones

**Comment by FudgePucker** [comment]
Hey remember when we all said boosting their speed to past max human speed would be a bad idea and too over powered?

**Comment by Megamaw** [comment]
overpowered for a race that can't consistently intake nutriment, and will almost constantly be hungry, balancing it out? for a race that has a few minor upsides, but awful downsides like no pills, no food except raw meat (so if the chef doesnt like you, you're fucked), no blood so viruses are almost completely incurable, die instantly to basically any brute damage, et cetera?

and the upsides are '1.2x faster when properly fed' and 'doesnt need to breathe'(which rarely matters, since ZAS will kill you instantly anyways).

sounds OK to me, personally.

either way, this isn't a buff to skeletons. if you're upset about skeletons being too strong, this is your nerf. it still keeps the current upside, but makes it less drastic.

**Comment by yclatious** [comment]
Stop trying to bullshit your way past the ack crack when nobody but you is to blame.

Acks have always been a shitty hardcore race and when you decided to buff them, everyone said you had no fucking reason or motive to do it but your own personal bias, the PR passed at a cost and everyone was talking about the compromise of them being buffed but also cracked, lo and behold, so it was.

Being honest and blunt, suck it the fuck up. You knew your buffs were horseshit, you knew that almost everyone agreed on if the buff passed for them to be removed from roundstart and you didnt at any point close it, at fucking all. You realized that was a serious probability and you didn't complain, least to my memory and knowledge, so you accepted that it was a outcome you were fine with and if you were too damn dense to understand that was the heavily acepted catch to the PR being passed, well, tough shit my lad, you are too dumb to complain when the thing was clearly being layed out on you, and it was several times and the sugestion to buff but remove was, effectively, said about 3 or 4 times.

Again, suck it the fuck up and deal with the fact your PR had no legitimacy and these PRs are nothing but a pathetic attempt at nerfing your shitty ack ack spouting meme race back on the game. Im not mad at skeletons being too strong, I was mad at them being a thing roundstart, I was mad at them being buffed, I was not mad at them being buffed but removed but I am somewhat miffed the guy who sucks skeleton dick so hard you could swear he is one himself is doing PRs to try and weasel his way out of shit, especialy after doing a stealth nerf to the ack probability under peoples noses.

So, in short, pathetic attempt at nerf so shitty meme race will be enabled roundstart again, for no reason, by a guy that only buffed them to begin with because he liked them and is now nerfing them because he got fucked by not understanding what so many people have said and agreed upon so the buff would pass. If he isnt getting git banned, atleast close his shitty meme race prs.

**Comment by Exxion** [comment]
Roundstart skeletons aren't coming back unless you can personally convince Pomf to bring them back
I recommend you take the buffs you can get and run with them

**Comment by Theophrastus1** [comment]
You reap what you sow. You got your buff @Megamaw you should be reveling in your victory.

**Comment by Kammerjunk** [comment]
I'm pretty sure it'll still go tits up the moment you have anything that changes your movement speed, anyway.

**Comment by Megamaw** [comment]
so let me get this straight

skeletons as is, as a result of the last PR, are far too powerful

therefore, the overwhelming consensus is to not nerf skeletons.

the reaction i've seen so far is assuming pomf is actually going to reinstate them. that's ultimately something i cannot influence.

but something i forgot to account for previously was that the 'melt' genetic power does more than just turn you into a skeleton, but actually fully heal you and turn you into a skeleton, even if you're dead. (unless that was a bug and was fixed)

skeletons are still in the game, and i feel like 'melt' currently would outclass all other genetic powers i can think of, which should probably not be the case.

**Comment by Theophrastus1** [comment]
Yeah, probably because that stupid buff you imposed to begin with. This is not a problem with skeletons, this is a problem with a buff that you bitched and moaned and fought to get merged, and it did. Now you're stating that you went too far and want to scale them back.

You don't even have the conviction to stand by your garbage choice here, and now you're pleading to us saying: "L-look I will reduce the speed a little so please put them back in please."

Come on @Megamaw just give a rest already. I mean, this is clearly a kneejerk reaction to you not getting what you entirely wanted here, and not a nerf based on consensus. Skellingtons got buffed as you wanted. However, in order to compensate for the drastically increased movement speed, pomf decided to disable them roundstart. That's the tradeoff. Unless pomf goes back on the decision, nothing you can do my man but nerf the skeletons that are now no longer roundstart thanks to your own self.

**Comment by Megamaw** [comment]
aye, i did say if it was considered 'too much' that i'd tone back the speed boost to 1.2 in the previous pr. whether they're roundstart or not, a consensus has been quickly met that it was way too far.

i'm already accepting that pomf is most likely not going to bring back skeletons. no, i'm not happy about it, but i'm a man of my word, and since pomf himself said it was too much, i'm aiming to dial it back, even if he straight up says in this PR they're not comin' back.

so no, i can't make pomf go back on his decision, or rewind time and make this have never happened. but that's not what i'm trying to do here. skeletons are still accessible in game, and i can live with that. but if people are saying 1.5 is too much, and it's still in the game accessible as is, it only makes sense to me to make good on my previous promise.

**Comment by Theophrastus1** [comment]
So then, essentially, you just caused precisely what you wanted to prevent. The Ack Crack

Sasuga, Megamaw

**Comment by Exxion** [comment]
So far, people have been complaining about this PR due to what it hopes to achieve (or at least appears to).
Can anyone tell me that they don't want this nerf, though, in a vacuum of context? Because if everyone's problem with it is just that they think Megamaw is doing it to get roundstart skeletons back, then that's not really a problem with the PR itself, and more a problem with previous events.

To be clear, yes, I'll merge it if nobody actually has anything against that.

**Comment by Theophrastus1** [comment]
@Exxion I feel that this is another kneejerk PR attempt at correcting his mistake of overbuffing the skeleton race. the buffs in place have hardly been in place long enough to determine whether they're far too strong for a non-roundstart race. I think that a little more time should pass before just reducing the speed on them otherwise, what was the point of the speed increase to begin with if he is already reducing it? He makes a valid point with the whole melt power in genetics, but I don't think I've seen many people taking advantage of that at least from the regulars in the discord since I haven't been able to play much the past few weeks.

Basically, I'd prefer if more time would pass considering another PR he put up about rebalancing his buff was closed fairly recently as well.

**Comment by kilozombie** [comment]
Skeletons are annoying to achieve now; the buff feels fine.

**Comment by yclatious** [comment]
Agreed with theo, this was far too damn quick to fully evaluate the actual effects on gameplay, shit reasoning aside, the relative rarity of skeletons and the people who will achieve them and fully try out, experiment and see if the buff is apropriate/give feedback makes it so this has to be documented, asked about and evaluated for a larger sum of time than to just kneejerk null the change.

Reasoning that are fully shit, asinine and made of personal bias and love of skeleton dick aside, and actually being fully serious and impartial, if you want to make a balance change, make sure it has a reason to exist and to be coded. This is a minimal time gap and we have not had a single bit of evidence YET as to this nerf being needed at all, show me times where it has been unbalanced enough to warrant a nerf and by all means go ahead and nerf it.

**Comment by Intigracy** [comment]
I'd rather have fast skeletons you get through genetics than slightly slower but still fast skeletons at roundstart.

**Comment by Blithering** [comment]
I'd rather a more thematic upside rather than just speed. Something like reattaching limbs without surgery

**Comment by Megamaw** [comment]
honestly yeah i don't know why i latched onto speed

i'd take normal speed + something actually interesting like purposeful strategic self-dismemberment than speedy bones any day if i could actually code it

**Comment by Blithering** [comment]
Ideaguy idea for skeleton racial thing: 
When they die, skeletons collapse into bone piles (like the ones left behind when someone is soulstoned), dropping all equipment. After a delay they can reassemble themselves and get back up. Taking the skull from the bone pile will prevent this from happening, but splashing blood on the skull will revive it as a living skull (monkey form of skellingtons). 
It's useful, thematic, but in a direct combat situation it would have little to no bearing as preventing the revive is a single click's worth of effort. In the event they can't be bothered to walk over to it and prevent the revive the skeleton still gets back up without any of their equipment on, so there's an additional delay before they can do anything in the way of returning to effectiveness. Cheesing space exploration could be prevented by putting a hefty cooldown or limited charges on the revive ability, and with skeletons being as brittle as they are they would still be inferior to diona at it.

**Comment by Megamaw** [comment]
honestly? i'm not a balancing expert, but that sounds pretty neat.

**Comment by kilkun** [comment]
>Acks have always been a shitty hardcore race and when you decided to buff them

no they weren't. skeletons having debuffs beyond "asimov doesn't have to listen to you" is a rather recent development.
